packages/data/src/api/tokenById/tokenById.ts

```diff
@@ -0,0 +1,47 @@
+import { fetchGraphQl } from "../../graphql/fetch";
+import { errorContractAddress, errorToken } from "./tokenById.mock";
+import { tokenByIdQuery } from "./tokenbyId.query";
+
+import { TokenByIdResults } from "./tokenById.types";
+
+export const checkParams = (tokenId: string, contractAddress: string) => {
+  if (typeof tokenId !== "string") {
+    return errorToken;
+  } else if (typeof contractAddress !== "string") {
```

Comment:
I think typescript should mostly help out here with the types and honestly, even if they were numeric graphql would figure this out.  Maybe we can just check for undefined and include message that like `did you forget to include contract address?`